modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/KnowledgeBaseArticleResourceTest.java

```diff
@@ -148,7 +195,7 @@ protected KnowledgeBaseArticle randomKnowledgeBaseArticle()
 			PortalUtil.getClassNameId(KBFolder.class.getName()), 0,
 			RandomTestUtil.randomString(), RandomTestUtil.randomString(),
 			RandomTestUtil.randomString(), RandomTestUtil.randomString(), null,
-			null, new Date(), null, null, null, serviceContext);
+			null, _datePublished, null, null, null, serviceContext);
```

Comment:
This is no longer fully random, as the same published date will be reused on different calls to `randomKnowledgeBaseArticle` on the same instance. Create a fresh random date every time the method is called.